website/docs/artist_getting_started.md

```diff
@@ -87,30 +92,35 @@ You can run AYON by desktop "P" icon (if it exists after installing) or by direc
 :::tip Is AYON running?
 AYON runs in the operating system's tray. If you see a green AYON icon in the tray you can easily tell AYON is currently running. Keep in mind that on Windows this icon might be hidden by default, in which case, the artist can simply drag the icon down to the tray.
 
-![Systray](assets/artist_systray.png)
+![A screenshot of the Windows System Tray showing the AYON icon](assets/artist_systray.png)
+:::
+
+:::tip Icon not showing on Linux
+Some Linux distributions do not ship with tray icons by defaults, mostly GNOME based, to get the icon, install the following extension: [Appindicator Support for GNOME Shell](https://extensions.gnome.org/extension/615/appindicator-support/)
 :::
 
 ## First Launch
 
 When you first start AYON, you will be asked to fill in some basic information.
 
-![login](assets/artist_login.png)
-
-### AYON Server URL
+![A screenshot of the AYON login dialog](assets/artist_login.png)
 
-You should have been supplied with AYON server url from your studio administrator. The URL looks something like; `https://ayon.dev/`.
+### AYON Login Details
 
-The `Username` and `Password` should also come from your studio administrator.
+Your Studio should provide you with the AYON `url`, `username` and `password` to fill in the dialog.
 
-### AYON Version Repository
+:::tip Example details
+URL: `https://ayon.dev/`
+username: `johndoe`
+password: `v3ry53cur3p455w0rd`
+:::
 
-Sometimes your Studio might also ask you to fill in the path to its version repository. This is a location where AYON will search for the latest versions, check if it's up to date and where updates are installed from automatically.
+If you want to try AYON and don't have a server, you can set up an account over at [AYON Cloud](https://ayon.cloud/)
 
-This path is usually taken from the database directly, so you shouldn't need it.
 
 ## Updates
 
-If you're connected to your Studio, AYON will check for, and install updates automatically every time you run it. That's why during the first start it can go through a quick update installation process, even though you might have just installed it.
+AYON updates automatically, based on your studio server, every time you launch it; that why on first time launch it might do a new installation, if your Studio is using a more recent version.on process, even though you might have just installed it.
```

Comment:
```suggestion
AYON updates automatically, based on your studio server, every time you launch it; that's why on first time launch it might do a new installation, if your Studio is using a more recent version, even though you might have just installed it.
```